fill-the-blanks/src/handler.py

```diff
@@ -105,29 +105,30 @@ def _customContentForCloze(self, txt, idx):
         return (txt, words)
 
 
-    def _extractTxt(self, input:str): 
+    def _extractTxt(self, inputWithHint): 
         try:
+            (input, hint) = inputWithHint
             content = BeautifulSoup('<span/>' + input, 'html.parser')
             text = ''.join(content.findAll(text=True))
-            return text
+            return (text, hint)
         except UserWarning:
             return input
```

Comment:
shouldn't it return a tuple also here?

Reply:
Yeah, should've returned `inputWithHint` too... that was careless of me